Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalDriver.h

```diff
@@ -115,7 +115,21 @@ typedef NS_ENUM(NSInteger, BTPayPalDriverErrorType) {
  @param completionBlock This completion will be invoked exactly once when checkout is complete or an error occurs.
 */
 - (void)requestBillingAgreement:(BTPayPalVaultRequest *)request
-                     completion:(void (^)(BTPayPalAccountNonce * _Nullable tokenizedPayPalAccount, NSError * _Nullable error))completionBlock;
+                     completion:(void (^)(BTPayPalAccountNonce * _Nullable tokenizedPayPalAccount, NSError * _Nullable error))completionBlock DEPRECATED_MSG_ATTRIBUTE("Use tokenizePayPalAccount instead.");
+
+/**
+ Tokenize a PayPal account for vault or checkout.
+
+ @note You can use this as the final step in your order/checkout flow. If you want, you may create a transaction from your
+ server when this method completes without any additional user interaction.
```

Comment:
Nitpick: The "if you want" language in the second sentence here seems weird to me. They should be creating a transaction after all checkout requests, maybe we can specify this further.

Reply:
I think there are cases where you wouldn't transact immediately after collecting payment information. That being said, I don't think this note adds a ton of valuable information, so maybe we should just remove it?